wix/WIXPatch.wxs

```diff
@@ -0,0 +1,10 @@
+<CPackWiXPatch>
+ <CPackWiXFragment Id="CM_CP_libraries.lib.afcuda.dll">
+  <Environment Id="ARRAYFIREPATH" Action="set"
+      Name="AF_PATH" Permanent="no" System="yes" Value="[INSTALL_ROOT]"/>
+  <Environment Id="ENVPATH" Action="set"
+      Name="PATH" Permanent="no" System="yes" Part="last" Value="[INSTALL_ROOT]lib"/>
+  <Environment Id="NVPATH" Action="set"
+      Name="PATH" Permanent="no" System="yes" Part="last" Value="[%CUDA_PATH]\nvvm\bin"/>
```

Comment:
This (adding NVVM to path) is something we have discussed at length internally about and have consistently decided not to have in our installer. The single most important reason being that NVIDIA does not add it to Path themselves.

Other reasons include
* Making the PATH too long.
* Most people will not remove such a path manually if they do not know what added it.